external-deps/spyder-kernels/.gitrepo

```diff
@@ -5,8 +5,8 @@
 ;
 [subrepo]
 	remote = https://github.com/spyder-ide/spyder-kernels.git
-	branch = remove_locals_inspection
-	commit = adb31866c64079bc023c4626f1e2cce3c51ab5f6
-	parent = d4fa5fd95e2ebe89cf49453ecd13cda9c72f73c2
+	branch = profile_script
```

Comment:
The branch in Spyder-kernels which has the new profile magics is called `improve_namespace`. So, please sync that branch here to test this PR manually.